datadog_checks_base/datadog_checks/base/checks/openmetrics/base_check.py

```diff
@@ -23,16 +23,28 @@ class OpenMetricsBaseCheck(OpenMetricsScraperMixin, AgentCheck):
 
     DEFAULT_METRIC_LIMIT = 2000
 
-    def __init__(self, name, init_config, agentConfig, instances=None, default_instances=None, default_namespace=None):
-        super(OpenMetricsBaseCheck, self).__init__(name, init_config, agentConfig, instances=instances)
+    def __init__(self, *args, **kwargs):
```

Comment:
This whole part seems quite hacky to keep compatibility, is something like this not doable?

```python
    def __init__(self, name, init_config, instances, default_instances=None, default_namespace=None):
        super(OpenMetricsBaseCheck, self).__init__(name, init_config, instances)
        self.default_instances = default_instances or {}
        self.default_namespace = default_namespace
```

And then update every integration extending this base class?

Reply:
No unfortunately because custom checks may depend on it